Remotion/Web/Development.WebTesting/ScreenshotCreation/Annotations/ScreenshotTooltipStyle.cs

```diff
@@ -27,7 +27,7 @@ namespace Remotion.Web.Development.WebTesting.ScreenshotCreation.Annotations
   /// </summary>
   public class ScreenshotTooltipStyle
   {
-    public static readonly ScreenshotTooltipStyle Chrome = new ScreenshotTooltipStyle (
+    public static readonly ScreenshotTooltipStyle Chromium = new ScreenshotTooltipStyle (

```

Comment:
Just a thought: Are we sure that the Tooltip Style is the same in Edge and Chrome?

Reply:
Yes